.github/workflows/uBench_whole_chain.yml

```
@@ -0,0 +1,90 @@
+name: Test the entire build and invokation chain of test examples.
+
+on: 
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  
+env:
+  GOOS: linux
+  GO111MODULE: on
+
+jobs:
+  compose_images:
+    name: Build and run chained functions serving
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+    - name: start multi-container program with docker-compose
+      run: |
+        cd ./function-images/tests/chained-function-serving/
+        docker-compose up -d
+        sleep 15s
+    - name: invoke the client
+      run: |
+        cd ./function-images/tests/chained-function-serving/
+        go run ./client/client.go
+        
+  knative_chain:
+    name: Start up and run the knative chain
+    env:
+        GITHUB_RUN_ID: ${{ github.run_id }}
+        LOGPATH: /tmp/uBench-logs/${{ github.run_id }}
+        GOCACHE: /root/tmp/gocache
+        GOPATH: /root/tmp/gopath
+    runs-on: [self-hosted, cri]
+    strategy:
+      fail-fast: false
+    steps:
+
+    - name: Set up Go 1.15
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: Setup LOGPATH
+      run: |
+        mkdir -p /tmp/uBench-logs/${{ github.run_id }}
+
+    - name: Host Info
+      run: |
+        echo $HOSTNAME
+        echo $GITHUB_RUN_ID
+        echo $LOGPATH
+
+    - uses: actions/checkout@v2
+
+    - name: start containerd
```

please unify the format in the file
```suggestion
    - name: Start containerd
```